docs/introduction/Ecosystem.md

```diff
@@ -2,7 +2,7 @@
 
 Redux is a tiny library, but its contracts and APIs are carefully chosen to spawn an ecosystem of tools and extensions.
 
-For an extensive list of everything related to Redux, we recommend [Awesome Redux](https://github.com/xgrommx/awesome-redux). It contains examples, boilerplates, middleware, utility libraries, and more.
+For an extensive list of everything related to Redux, we recommend [Awesome Redux](https://github.com/xgrommx/awesome-redux). It contains examples, boilerplates, middleware, utility libraries, and more.  [React-Redux Links](https://github.com/markerikson/react-redux-links) contains useful resources for anyone learning React or Redux, and [Redux Ecosystem Links](https://github.com/markerikson/redux-ecosystem-links) lists many Redux-related libraries and addons.
```

Comment:
Two nits:

* Let’s avoid double spaces in the source? I know they are stripped from the output but the inconsistency is still a little jarring. Sentences should be separated by a single space.

* `React-Redux` is a bit jarring because we never call it like this in the documentation. Can we change this to `React Redux Links`?

Thanks!